### PR TITLE
371 planning collect information about interface between planning and acting

### LINKED ISSUE #371: Planning: Collect Information about Interface between Planning and Acting
### Detailed Description

How does the communication between the planning and the acting part work? Are there any problems in the current solution?

### Definition of Done

Collected information is added as a document in the research folder.

### Effort Estimate

_No response_

### Testability

_No response_

### Dependencies

_No response_

### PR DESCRIPTION
**Pull request by Vroni27**
# Description
Markdown document including graphics to visualize the communication between the planning and acting components is added to the research folder.

Fixes #371

## Type of change

- Added Information in the research folder

## Does this PR introduce a breaking change?

No

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works (might be obsolete with CI later on)
- [x] New and existing unit tests pass locally with my changes (might be obsolete with CI later on)

## Summary by CodeRabbit

- **New Features**
	- Introduced a comprehensive overview of communication dynamics between planning and acting components in robotics.
	- Added sections detailing useful commands and message types for key communication topics.
	- Included diagrams and examples to enhance understanding of message structures and practical applications.

### CODE REVIEW

File: doc/research/paf24/planning_acting_relation.md
Comment (johannaschwarz) [inline doc/research/paf24/planning_acting_relation.md:1]: sollen wir einen unterordner /planning erstellen?

File: doc/research/paf24/trajectory_example.png
Comment (johannaschwarz) [inline doc/research/paf24/trajectory_example.png:1]: die assets vielleicht doch in den doc/assets/research_assets ordner verschieben

**Review by johannaschwarz — APPROVED** [review]

File: doc/research/paf24/planning_acting_relation.md
Comment (Vroni27) [reply doc/research/paf24/planning_acting_relation.md:1]: Ja, ich verschiebe alles in den Planning Unterordner

File: doc/research/paf24/trajectory_example.png
Comment (Vroni27) [reply doc/research/paf24/trajectory_example.png:1]: Ja, ich habe sie verschoben

**Comment by johannaschwarz** [comment]
Die Bilder fehlen jetzt

**Pull request merged**